src/models/Call-types.ts

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2024 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+export const JitsiCallMemberEventType = "io.element.video.member";
+
+export interface JitsiCallMemberContent {
```

Comment:
are we duplicating these? they still seem to exist in `src/call-types.ts`

Reply:
Right you are, same cause as the other thing